scripts/kb_scripts/gwf_to_scs.py

```diff
@@ -0,0 +1,120 @@
+import argparse
```

Comment:
Will same script in ims.ostis.kb be removed later? Otherwise we'll have duplicated kb scripts in 2 places

Reply:
Currently it's present in ostis-web-platform, but there's https://github.com/ostis-ai/ostis-web-platform/pull/8 that deletes the scripts from the platform. There will be no duplicates.